### PR TITLE
CO-2637_Email_for_petitions_Pending_Approval_should_be_configurable

### PR DESCRIPTION
**Pull request by ioigoume**
Approver Notifications Message Templates

### CODE REVIEW

File: app/Controller/CoEnrollmentFlowsController.php
Comment (boshrin) [inline app/Controller/CoEnrollmentFlowsController.php:162]: Do we need a new context for this? It's still an Approval related email, we could probably just use `EnrollmentApproval`. If we do want a new context, it should follow the existing naming pattern, eg `EnrollmentApprover`.

File: app/Model/CoMessageTemplate.php
Comment (boshrin) [inline app/Model/CoMessageTemplate.php:327]: I'm not sure where these default values came from (I guess I missed the PR that added them) but we shouldn't be using these defaults as they might confuse admins who receive them. If we need a default text, it should just be "(unknown)" or "(?)" to indicate that something went wrong.

File: app/Model/CoPetition.php
Comment (boshrin) [inline app/Model/CoPetition.php:2444]: Any reason to commit a commented out block? If so there should be a note.

**Review by boshrin — CHANGES_REQUESTED** [review]

File: app/Model/CoMessageTemplate.php
Comment (ioigoume) [reply app/Model/CoMessageTemplate.php:327]: These default values are used only when testing a template. They do not interfere with the rest of the flow.

Comment (ioigoume) [reply app/Model/CoMessageTemplate.php:327]: These default values are used only when testing a template. They do not interfere with the rest of the flow. The default, dummy value is needed because when testing the template there are no meaningful data available.

File: app/Model/CoPetition.php
Comment (ioigoume) [reply app/Model/CoPetition.php:2444]: The latest commit does not include this block of code

File: app/Controller/CoEnrollmentFlowsController.php
Comment (ioigoume) [reply app/Controller/CoEnrollmentFlowsController.php]: I renamed the enum variable from ApproverNotification to EnrollmentApprover. The Approver template has its own foreign key. As a result i followed the pattern already applied for the other types of Templates. As far as i am concerned and it is more clear to keep them separate.

File: app/Model/CoMessageTemplate.php
Comment (boshrin) [reply app/Model/CoMessageTemplate.php:327]: ok. the default values for testing should be defined somewhere closer to the testing code then, but that's not a topic for this PR.

**Review by boshrin — APPROVED** [review]

**Pull request merged**
